### DISCUSSION TITLE
MCX Cloud service is temporarily offline

### DISCUSSION THREAD

**Discussion by fangq** [fangq/mcx#170] [Announcements]
The past Monday, due to a university network upgrade, the backend server running the MCX Cloud services became offline. I became aware of this issue yesterday but was not able to find time reboot the server. 

I will check the server tomorrow and hopefully a reboot should resolve the issue. will post update once the service is back online.

**Comment by fangq** [comment]
FYI, the mcx cloud server has been rebooted and it appears that the service has been restored.

**Reply by zqh351154** [reply]
服务器好像被关闭了，一直处于排队进程，无法运行
